src/constants/translations/en/login.json

```diff
@@ -15,4 +15,4 @@
   "resetPasswordDesc": "Your new password must be different from previous used passwords.",
   "savePassword": "Save password",
   "sucsessReset": "Your password has been successfully updated!"
```

Comment:
Could you please find all instances of the variable "sucsessReset" and change them to "successReset"?

Reply:
Is this a part of current issue? Maybe would be better to open another one?

Reply:
Agree with creating a new issue, , has anyone opened it?

Reply:
I will open a new issue on it